site/src/components/PageSidebar.tsx

```diff
@@ -381,24 +383,34 @@ export const Sidebar: VFC<SidebarProps> = ({
         }}
       />
       <Box
-        paddingRight={3}
         sx={{
           maxHeight: isSticky ? "100vh" : undefined,
-          overflow: isSticky ? "auto" : undefined,
```

Comment:
Can we not retain something like this, or move it to the `Box` below?

Not a fan of having `overflow: scroll` on permanently if we can avoid it

Reply:
Not sure why, but I did not see a scrollbar on tall screens depsite `overflow: "scroll"`. I changed it to `"auto"` just in case. If you still see some oddities in the UI, could you please share a screenshot? Happy to take another look.

Reply:
Looks good now, thanks!